SpectralLine/SpectralLineCrawler.cc

```diff
@@ -20,6 +20,14 @@ const std::string SpectralLineCrawler::Headers[] = {"Species", "Chemical Name",
     "Unresolved Quantum Numbers", "CDMS/JPL Intensity", "S<sub>ij</sub>&#956;<sup>2</sup> (D<sup>2</sup>)", "S<sub>ij</sub>",
     "Log<sub>10</sub> (A<sub>ij</sub>)", "Lovas/AST Intensity", "E_L (cm^-1)", "E_L (K)", "E_U (cm^-1)", "E_U (K)", "Linelist"};
 
+const std::map<std::string, std::string> SpectralLineCrawler::HeaderTypeMap = {{"Species", "string"}, {"Chemical Name", "string"},
```

Comment:
I would suggest formatting this with each key/value pair on a new line -- this style of formatting appears in `Hdf5Loader.cc`, so it should be allowed by `clang-format`.

Also, this should be an `unordered_map`.